smart/Conditional.md

```diff
@@ -0,0 +1,35 @@
+# Conditional balances
+
+## Article I: OPEN PROBLEMS
+
+An _Open Problem_ is a problem which has no answer _yet_. Examples of _Open Problems_ include:
+- Is P == NP ?
+- Will the sun rise tomorrow ?
```

Comment:
```suggestion
- Navier–Stokes existence and smoothness
```
The question you proposed can be solved by induction so it's not an _open_ question, sorry.

Reply:
I tend to disagree. Counter-proof: at some point the Sun explodes so it might not rise tomorrow.

Reply:
Yes but this point is available at a n point in the future :sun_with_face:

Reply:
I now understand your point. Indeed the case is solved. Sun will explose so question not _Open_.